sdks/java/io/file-based-io-tests/src/test/java/org/apache/beam/sdk/io/common/FileBasedIOTestPipelineOptions.java

```diff
@@ -48,4 +48,15 @@
   boolean getReportGcsPerformanceMetrics();
 
   void setReportGcsPerformanceMetrics(boolean performanceMetrics);
+
+  @Validation.Required
+  @Description("Expected hash for write correctness checking")
+  String getExpectedHash();
+
+  void setExpectedHash(String hash);
+
+  @Description("Size of data saved on the target filesystem")
+  Integer getDatasetSize();
```

Comment:
I agree that this option should not be required but what if it's null? Will the metric suppliers still save it as null or will the test fail?

Reply:
What is the unit of size we use in this option (I assume that it's bytes)? This should also be documented.

I propose: `Size of data saved on the target filesystem (bytes)`

Reply:
Besides that: this should probably eventually land in IOTestPipelineOptions interface but since you are working on file Based tests only in this PR I'm fine with the current state. This is just a note for the future. :)